test_scripts/Defects/5_0/2400_SDL_behavior_in_case_mobile_app_sends_reques_with_at_least_one_choiseSet_without_vrCommands_parameter_and_at_least_one_choiseSet_with_vrCommands_at_MANUAL_ONLY_mode.lua

```diff
@@ -0,0 +1,102 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2400
+--
+-- Description:
+-- SDL behavior in case mobile app sends reques with at least one choiseSet without parameter and at least one choiseSet with at MANUAL_ONLY mode 
+-- Steps to reproduce:
+-- 1) Mobile app sends PerformInteraction with interactionMode: MANUAL_ONLY and one or more (choiseSet_created_ without and one
+--    or more choiseSet_created with) parameters.
+-- Expected:
+-- 1) Send VR.PerformInteraction with all prompts (helpPrompt, timeoutPrompt, initialPrompt) to HMI
+-- 2) Send UI.PerformInteraction with all requested choiseSets to HMI
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+-- [[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+-- [[ Local variables ]]
+local hmiParams = {
+    helpPrompt = { type = "TEXT", text = "helpPrompt" },
+    initialPrompt = { type = "TEXT", text = "initialPrompt" },
+    timeoutPrompt = { type = "TEXT", text = "timeoutPrompt" }
+}
+
+local params = {
+    initialText = "StartPerformInteraction",
+    interactionMode = "MANUAL_ONLY",
+    interactionChoiceSetIDList = { 100 },
```

Comment:
@TMyza Please avoid to create hardcode dependencies between variables with different visibility scopes.
Example of mentioned dependency from current script:
```
(local scope)params.interactionChoiceSetIDList <==> (function createInteractionChoiceSetVR scope)choiceParams.interactionChoiceSetID
```

Also it is existing overriding of table field `interactionChoiceSetIDList` value here: initially this field has value `{ 100 }` and it is overridden with value `{ 110 }` in next line. Please fix this.